app/controllers/show-project.js

```diff
@@ -61,6 +61,14 @@ export default class ShowProjectController extends Controller {
     });
   }
 
+  @computed('model.bbl_featurecollection')
+  get hasBBLFeatureCollectionGeometry() {
+    const geom = this.model.bbl_featurecollection.features[0].geometry;
```

Comment:
What does it look like in cases where there is no bbl geom? I ask because a feature collection can be empty:

```javascript
{
  type: 'FeatureCollection',
  features: [],
}
```

In that case, you would get an unhandled exception: "cannot find property 'geometry' of undefined'". 

If we look at the model attr for bbl_featurecollection, there's no default: https://github.com/NYCPlanning/labs-zap-search/blob/develop/app/models/project.js#L78. 

So we can't guarantee it to be not null. Thoughts? 

I think if it were me, I would do it like this:

```javascript
@computed('model.bbl_featurecollection')
get hasBBLFeatureCollectionGeometry() {
  return this.model.bbl_featurecollection.features.length;
}
```

This could return 0, which evaluates to "falsey" in the template, or some number greater than 0, which will be "truthy" in the template.

Then, I would go here to set up a default so we can guarantee that the model is always some kind of feature collection:
https://github.com/NYCPlanning/labs-zap-search/blob/develop/app/models/project.js#L78

You can see an example of this here: https://github.com/NYCPlanning/labs-applicantmaps/blob/develop/app/models/geometric-property.js#L107-L108.

Happy to pair on this!

Reply:
Awesome, thank you! Confused about how this change covers the case where a project DOES have a feature collection BUT the geometry is `null`? Because in the case that I'm thinking of, a feature collection exists but only the geometry is `null`, meaning the computed above would not cover that situation, because the `length` would still come back as greater than 0. 

This is currently happening in the new Gowanus project, because BBL geometry has not been loaded yet for this project, but it DOES have a feature collection. <--- this might be a bug, so maybe this situation should never happen, and if that's the case then `this.model.bbl_featurecollection.features` would only exist for projects that for sure had `geometry` not equal to `null`

Reply:
@trbmcginnis gotcha, I see! A null geometry is also definitely compliant with the GeoJSON spec, too.

In that case, I would add your logic back! -> 

```javascript
return this.model.bbl_featurecollection.features.length 
  && this.model.bbl_featurecollection.features[0].geometry;
```

I'm not sure why but you need the length check before you check for the geometry to avoid situations with empty features. So, I still think we need a default value for the model. Thoughts?

Reply:
Sweeettt, I'll go ahead and make that change :).

Yes, I agree that adding a default value to the model is a good idea--anything to make it more secure

Reply:
Changed!